editor/example/index.ts

```diff
@@ -443,17 +447,10 @@ export const editorExampleObj: {
         spec: JsonExampleSpecs.EX_SPEC_SPATIAL_MULTIPLE_TRACKS,
         image: THUMBNAILS.SPATIAL_DRAFT
     },
-    SPATIAL_LAYOUT_GRAMMAR_EXTENSION: {
-        group: 'Experimental',
-        name: 'Spatial Layout: Grammar Extension',
-        spec: JsonExampleSpecs.EX_SPEC_SPATIAL_LAYOUT_GRAMMAR_EXTENSION,
-        image: THUMBNAILS.SPATIAL_DRAFT,
-        forceShow: true
-    },
```

Comment:
Removed this example I previously created since it is very similar to another one (`SPATIAL_LAYOUT_3D_YEAST_MODEL`).